mergify_engine/actions/merge/action.py

```diff
@@ -181,16 +239,27 @@ def _merge(self, ctxt):
                 "",
             )
 
-        commit_title_and_message = self._get_commit_message(
-            ctxt.pull, self.config["commit_message"]
-        )
-        if commit_title_and_message is None:
-            data = {}
-        else:
-            data = {
-                "commit_title": commit_title_and_message[0],
-                "commit_message": commit_title_and_message[1],
-            }
+        data = {}
+
+        try:
+            commit_title_and_message = self._get_commit_message(
+                ctxt.pull_request, self.config["commit_message"],
+            )
+        except UndefinedVariables as e:
+            return (
+                "failure",
```

Comment:
You can use `action_required` here. This makes the engine retries and put a nice icon in check list.